api/config.js

```diff
@@ -20,20 +23,26 @@ if (!isProduction && !isDevelopment) {
 
 if (!secret) {
 	if (isProduction) {
-		logger.error("the secret was not provided");
 		logger.error("set HACKILLINOIS_SECRET to a secure, random string");
-
 		process.exit(1);
 	}
 
 	secret = 'NONE';
 }
 
+if (!mailApiKey) {
+	if (isProduction) {
+		logger.error("set HACKILLINOIS_MAIL_KEY to the API key");
+		process.exit(1);
+	} else {
+		logger.warn("the mail server API key was not set");
+		logger.warn("some mail service functionality may malfunction");
+	}
+}
+
 if (!superuserEmail) {
```

Comment:
Were you thinking verification by format, or actually making sure that the email exists?